docs/UserGuide.adoc

```diff
@@ -711,26 +711,37 @@ Collapses the folder hierarchy by 1 level.
 === Switching <<Tabs,tabs>>: *`tab`*
 
 Switches the current app view to the tab that corresponds to the given
-index.
+index. For your convenience, you can use keywords `dash`, `on` or `off` instead of an index.
 
-Format: `*tab* INDEX`
+Format: `*tab* INDEX` OR `*tab* KEYWORD`
 
 ****
 * `INDEX` is `1`, `2` or `3`, which correspond to the
 <<Dashboard,Dashboard>>, Online, and Offline tabs respectively.
+* `KEYWORD` is `dash`, `on` or `off`, which correspond to the
+<<Dashboard,Dashboard>>, Online, and Offline tabs respectively.
 ****
 
 Examples:
```

Comment:
Can we select just 2 or 3 of the examples to keep the UG concise? Maybe `tab dash` and `tab 3` (and possibly `tab on`, if we want to explain all the different tabs)